src/main/java/org/javarosa/core/model/instance/geojson/GeojsonGeometry.java

```diff
@@ -17,23 +17,44 @@
 package org.javarosa.core.model.instance.geojson;
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
+
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.List;
+import java.util.StringJoiner;
 
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 public class GeojsonGeometry {
-    private String type;
-    private ArrayList<String> coordinates;
 
-    public String getType() {
-        return type;
-    }
+    private String type;
+    private ArrayList<Object> coordinates;
```

Comment:
I don't think there's a better way to parse properties that can have multiple types, unfortunately.